app/controllers/storage_tables/disk_controller.rb

```diff
@@ -6,8 +6,21 @@ module StorageTables
   # Always go through the BlobsController, or your own authenticated controller, rather than directly
   # to the service URL.
   class DiskController < BaseController
+    include StorageTables::FileServer
+
     skip_forgery_protection
 
+    def show
+      if encoded_checksum
+        serve_file StorageTables::Blob.service.path_for(encoded_checksum[:checksum]),
```

Comment:
Dit werkt enkel voor de disk service? Willen we dit wel in gebruik nemen nu we juist van de disk service af willen?